content/en/agent/basic_agent_usage/aix.md

```diff
@@ -84,25 +84,25 @@ To enable the above integrations, copy and edit the sample configuration files p
 
 DogStatsD allows collecting and submitting custom metrics to Datadog. It listens on a UDP port and StatsD metrics may be submitted to it. These are then relayed to Datadog.
 
-DogStatsD relies on the same configuration file defined for the agent and runs in a separate process. To enable DogStatsD do:
+DogStatsD relies on the same configuration file defined for the Agent and runs in a separate process. To enable DogStatsD:
```

Comment:
The entire dogstatsd section is outdated and should read:


> Dogstatsd allows collecting and submitting custom metrics to datadog. It listens on
a UDP port and statsd metrics may be submitted to it. These will then be relayed
to Datadog.

> Dogstatsd relies on the same configuration file defined for the agent, where a `dogstatsd`
configuration section is available. The dogstatsd server will typically run within the
same agent process, but should you need a dedicated process it may also be launched in
standalone mode.

> To enable dogstatsd, simply edit `/etc/datadog-agent/datadog.yaml` and set the relevant
configuration options.

```yaml
dogstatsd:                        # Dogstatsd configuration options
  enabled: true                   # disabled by default
  bind_host: localhost            # address we'll be binding to
  port: 8125                      # dogstatsd UDP listening port
  non_local_traffic: false        # listen to non-local traffic
```